src/payments/services/payment.service.ts

```
@@ -46,6 +50,89 @@ export class PaymentService {
+      );
+    }
+
+    let total = 0;
+    let pricingCurrency: string | undefined;
+
+    for (const courseId of uniqueCourseIds) {
+      const pricing = await this.lmsClientService.getCoursePricing(
+        courseId,
+        tenantId,
+        organisationId,
+        academicYearId,
```

ESLint

[error] Delete `,`

(prettier/prettier)